Opsero Electronic Design Inc. 2021
Jeff Johnson
Remove PXE patch 2020.2
=======================
In 2020.2 there is a U-Boot build error on Microblaze designs that don't have
Ethernet. The problem is described here:
https://forums.xilinx.com/t5/Embedded-Linux/Petalinux-2020-2-build-failure-Microblaze-without-Ethernet/td-p/1181581
This patch prevents PXE from being pulled into the build regardless of the config.

diff --git a/cmd/Makefile b/cmd/Makefile
index 3feb7741c8..d0d9bc295d 100644
--- a/cmd/Makefile
+++ b/cmd/Makefile
@@ -110,7 +110,6 @@ ifdef CONFIG_PCI
 obj-$(CONFIG_CMD_PCI) += pci.o
 endif
 obj-$(CONFIG_CMD_PINMUX) += pinmux.o
-obj-$(CONFIG_CMD_PXE) += pxe.o pxe_utils.o
 obj-$(CONFIG_CMD_WOL) += wol.o
 obj-$(CONFIG_CMD_QFW) += qfw.o
 obj-$(CONFIG_CMD_READ) += read.o
@@ -129,7 +128,6 @@ obj-$(CONFIG_CMD_SETEXPR) += setexpr.o
 obj-$(CONFIG_CMD_SPI) += spi.o
 obj-$(CONFIG_CMD_STRINGS) += strings.o
 obj-$(CONFIG_CMD_SMC) += smccc.o
-obj-$(CONFIG_CMD_SYSBOOT) += sysboot.o pxe_utils.o
 obj-$(CONFIG_CMD_TERMINAL) += terminal.o
 obj-$(CONFIG_CMD_TIME) += time.o
 obj-$(CONFIG_CMD_TRACE) += trace.o
